designs/manager_policy_mapping/policy_mapping.md

```diff
@@ -1,13 +1,13 @@
-# Policy Mapping Workflow with IP/MAC
+# Multi-Policy Mapping Workflow
 
-This document describes the proposed changes for policy mapping workflow with IPs/MACs.
+This document describes the proposed changes for multi-policy mapping workflow.
 
 ## Background
 
 Currently, we have two ways to determine which policy gets applied to which workload:
 
 - Provide the VNF security manager with encapsulation tag
-- Provide the VNF security manager with IP/MAC
+- Provide the VNF security manager with list of policies
```

Comment:
IP/Mac is still relavent. 

Reword to -Provide the VNF security manager with IP/Mac and a list of polices/tags to associate with them